samples/perf_client/perf_client.h

```diff
@@ -379,6 +391,12 @@ namespace client
       size_t read;
       size_t written;
 
+      if (options.transactions_per_s > 0) {
+        write_delay_us = 1000000 / options.transactions_per_s;
+        connection->set_tcp_nodelay(true);
```

Comment:
We should set `TCP_NODELAY` in all cases, outside of this check. Even without a target transaction rate, this should make the performance tests more predictable.

Reply:
In throughput tests, I don't expect it will produce a measurable effect, we are writing heavily.